Sources/Common/ConnectionHandler.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ConnectionHandler.swift
+//  InstantSearchCore
+//
+//  Created by Vladislav Fitc on 26/11/2019.
+//  Copyright © 2019 Algolia. All rights reserved.
+//
+
+import Foundation
+
+public class ConnectionHandler: Connection {
```

Comment:
I'm not sure if this class is used somewhere. If not let's remove it

Reply:
Done